.travis.yml

```diff
@@ -1,6 +1,28 @@
+sudo: required
+dist: trusty
+
 language: node_js
 node_js:
   - "4.1"
+python:
+  - "3.4"
+
+cache:
+  pip: true
+  directories:
+    - node_modules
```

Comment:
I don't know if this works as expected ... (Not wure what the current directory is when running this)
~/coala-html/node_modules (or whatever it is really) would be better